LBHackney-IT/lbh-fss-portal-frontend
# Feature/add analytics page & spellcheck

**Description**

- added analytics page which shows a summary of the number of organisations and services 
- added spell check to various fields 

**Test instructions**
- pull this branch 
- `yarn install`
- **Analytics page** 
    - navigate to **/analytics** and check numbers are displayed
    - select a different date from the dropdown and check numbers updates accordingly (the mocked data has 'created_at' dates in early 2020 so try setting the date filter to weeks in Jan 2020 to see numbers update, then try setting the date filter to later in 2020 to see 0's) 
    - compare against figma designs (see below)

- **Spell check** 
    - check that spellcheck has been added to relevant input fields on organisation and service forms (the task specifies spellcheck should be added to all text fields, but i think there are many text fields this should not apply to e.g. providing links to external sites, providing user names, organisation names, charity reference number, etc... I have therefore used my judgement as to what is relevant) 
    - navigate to **/organisations/75720/edit**, proceed through the form and check spellcheck is functional and if there are any additional fields you think spell check should be added to 
    - navigate to **/services/add**, proceed through the form and check spellcheck is functional and if there are any additional fields you think spell check should be added to 


figma designs: 
https://www.figma.com/file/t6fanDKvLkCHJsY85ivFTF/Services-Directory?node-id=598%3A44959

![image](https://user-images.githubusercontent.com/69238887/95025638-31262d00-0683-11eb-9e44-0f9c6a289268.png)

**Tickets**
https://app.clickup.com/t/9e5u8n
https://app.clickup.com/t/9e5u9a
